support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.tsx

```diff
@@ -337,7 +337,7 @@ function WeeklyCheckoutForm(props: PropTypes) {
 					<BillingPeriodSelector
 						fulfilmentOption={props.fulfilmentOption}
 						onChange={(billingPeriod) => props.setBillingPeriod(billingPeriod)}
-						billingPeriods={weeklyBillingPeriods()}
+						billingPeriods={weeklyBillingPeriods}
```

Comment:
I couldn't quite figure out why this changed, but assuming it's okay (it used to be pass the result of calling the method, but now we're passing the method by name? I couldn't see an equivalent to this in the `BillingPeriodSelector` component).

Reply:
`weeklyBillingPeriod` is defined here, was a function then a const to bring  it into line with the gifting one -> 
![image](https://github.com/user-attachments/assets/83e9f560-01c4-4703-8716-e8b16ef5ec2b)